Docs/sphinx_documentation/source/External_Profiling_Tools.rst

```diff
@@ -279,3 +279,167 @@ generated figures. Some examples are shown here.
 
 .. [5]
    https://www.nersc.gov/users/software/performance-and-debugging-tools/ipm/
+
+NSight Systems
+==============
+
+The NSight Systems tool provides a high-level overview of a GPU code, displaying the kernel
+launches, API calls, NVTX regions and more in a timeline for a clear, visual picture of the
+overall runtime patterns.  It analyzes CUDA-based GPU codes and is available on Summit and
+Cori in a system module.
+
+NSight Systems provides a variety of profiling options.  This documentation will cover the
+most commonly used options for AMReX users to keep track of useful flags and analysis
+patterns. For the complete details of using NSight Systems, refer to the official documentation.
+
+Profile Analysis
+----------------
+
+The most common use case of NSight Systems for AMReX users is the creation of a qdstrm file
```

Comment:
The creation of a `qdrep` file is the goal. The `qdstrm` file is an intermediate internal representation which should not be used by users in recent versions of Nsight Systems.